Makefile

```diff
@@ -68,8 +68,10 @@ test-e2e-18-local: setup
 	LOCAL_OPERATOR=true JBOSS_HOME=/wildfly OPERATOR_NAME=wildfly-operator ./operator-sdk test local ./test/e2e/18.0 --verbose --debug  --namespace default --up-local
 
 ## test-e2e-18           Run e2e test for WildFly 18.0 with a containerized operator
+test-e2e-18: export component := wildfly-operator
```

Comment:
for reference, could you please link to the CI job that uses that target?

I wonder if we should make a distinct target for it (so that the test-e2e-18 target can also be run locally by developers). something like `test-e2e-ci`?